firmware-support/bittide-sys/src/uart.rs

```diff
@@ -10,6 +11,7 @@ pub struct UartStatus {
 pub struct TransmitBufferFull;
 pub struct ReceiveBufferEmpty;
 
+#[derive(Clone, Copy)]
```

Comment:
Does this need to have `Copy`? I think `Clone` is fine, but `Copy` makes it really easy to create new instances when maybe that's not actually intended.

Reply:
Makes sense :)